web/src/components/DatePicker.tsx

```diff
@@ -75,16 +41,22 @@ const isValidJsDate = (date: Date | null): boolean => {
   if (date == null) {
     return false
   }
-  try {
-    const checkDate = DateTime.fromJSDate(date)
-    const maxValidYear = 2500
-    // Added this operator to prevent crashing rrule when typing random number
-    return checkDate.year <= maxValidYear
-  } catch (_) {
-    return false
+  const checkDate = DateTime.fromJSDate(date)
+  const maxValidYear = 2500
+  return checkDate.year <= maxValidYear
+}
+
+export class CustomAdapterLuxon extends AdapterLuxon {
+  parse = (value: string, format: string): DateTime | null => {
+    try {
+      return DateTime.fromFormat(value, format)
+    } catch (_) {
+      return null
+    }
   }
+
+  getInvalidDate = (): DateTime => DateTime.now()
```

Comment:
❓ Is the implementation of `getInvalidDate` also necessary? It seems to work fine for me without.